diff --git a/indi-armadillo-platypus/CMakeLists.txt b/indi-armadillo-platypus/CMakeLists.txt
index c5ec88d2..7114a966 100644
--- a/indi-armadillo-platypus/CMakeLists.txt
+++ b/indi-armadillo-platypus/CMakeLists.txt
@@ -6,7 +6,7 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
 SET(CMAKE_CXX_STANDARD 11)
-SET(RULES_INSTALL_DIR "/lib/udev/rules.d/")
+SET(RULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d/")
 
 find_package(INDI REQUIRED)
 
diff --git a/indi-dsi/CMakeLists.txt b/indi-dsi/CMakeLists.txt
index 24569a7c..66509b0f 100644
--- a/indi-dsi/CMakeLists.txt
+++ b/indi-dsi/CMakeLists.txt
@@ -8,8 +8,8 @@ include(GNUInstallDirs)
 IF(APPLE)
 set(FIRMWARE_INSTALL_DIR "/usr/local/lib/indi/DriverSupport/dsi")
 ELSE(APPLE)
-set(RULES_INSTALL_DIR "/etc/udev/rules.d")
-set(FIRMWARE_INSTALL_DIR "/lib/firmware")
+set(RULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/etc/udev/rules.d")
+set(FIRMWARE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/firmware")
 ENDIF()
 set (DSI_VERSION_MAJOR 0)
 set (DSI_VERSION_MINOR 3)
diff --git a/indi-ffmv/CMakeLists.txt b/indi-ffmv/CMakeLists.txt
index b2292e20..a109ab07 100644
--- a/indi-ffmv/CMakeLists.txt
+++ b/indi-ffmv/CMakeLists.txt
@@ -7,7 +7,7 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
 IF(NOT APPLE)
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 ENDIF(NOT APPLE)
 
 find_package(CFITSIO REQUIRED)
diff --git a/indi-fli/CMakeLists.txt b/indi-fli/CMakeLists.txt
index 656e72d3..da2148c1 100644
--- a/indi-fli/CMakeLists.txt
+++ b/indi-fli/CMakeLists.txt
@@ -5,7 +5,7 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
-set (UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set (UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 find_package(CFITSIO REQUIRED)
 find_package(INDI REQUIRED)
diff --git a/indi-gphoto/CMakeLists.txt b/indi-gphoto/CMakeLists.txt
index f8a2e398..82f830d4 100644
--- a/indi-gphoto/CMakeLists.txt
+++ b/indi-gphoto/CMakeLists.txt
@@ -8,7 +8,7 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 SET(BIN_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/bin")
 SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall" )
diff --git a/indi-mi/CMakeLists.txt b/indi-mi/CMakeLists.txt
index e9b6eb53..954ee494 100644
--- a/indi-mi/CMakeLists.txt
+++ b/indi-mi/CMakeLists.txt
@@ -5,7 +5,7 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 find_package(INDI REQUIRED)
 find_package(USB1 REQUIRED)
diff --git a/indi-nightscape/CMakeLists.txt b/indi-nightscape/CMakeLists.txt
index b1641003..c328eb3a 100644
--- a/indi-nightscape/CMakeLists.txt
+++ b/indi-nightscape/CMakeLists.txt
@@ -97,7 +97,7 @@ install(TARGETS indi_nightscape_ccd RUNTIME DESTINATION bin )
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/indi_nightscape.xml DESTINATION ${INDI_DATA_DIR})
 
 if (${CMAKE_SYSTEM_NAME} MATCHES "Linux")
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/99-nightscape.rules DESTINATION ${UDEVRULES_INSTALL_DIR})
 endif()
 
diff --git a/indi-qsi/CMakeLists.txt b/indi-qsi/CMakeLists.txt
index e1f7f66a..ed32a5cc 100644
--- a/indi-qsi/CMakeLists.txt
+++ b/indi-qsi/CMakeLists.txt
@@ -6,7 +6,7 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
 SET(CMAKE_CXX_STANDARD 11)
-SET(RULES_INSTALL_DIR "/lib/udev/rules.d/")
+SET(RULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d/")
 
 find_package(CFITSIO REQUIRED)
 find_package(INDI REQUIRED)
diff --git a/indi-ssag/CMakeLists.txt b/indi-ssag/CMakeLists.txt
index a990ae60..0f46a344 100644
--- a/indi-ssag/CMakeLists.txt
+++ b/indi-ssag/CMakeLists.txt
@@ -8,7 +8,7 @@ include(GNUInstallDirs)
 find_package(INDI REQUIRED)
 find_package(USB1 REQUIRED)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 set(INDI_SSAG_VERSION_MAJOR 1)
 set(INDI_SSAG_VERSION_MINOR 0)
diff --git a/indi-sx/CMakeLists.txt b/indi-sx/CMakeLists.txt
index cddc71a0..506c4ee4 100644
--- a/indi-sx/CMakeLists.txt
+++ b/indi-sx/CMakeLists.txt
@@ -4,7 +4,7 @@ PROJECT(indi_sx CXX C)
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
-set(RULES_INSTALL_DIR "/lib/udev/rules.d/")
+set(RULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d/")
 
 SET(CMAKE_CXX_STANDARD 11)
 set (VERSION_MAJOR 1)
