diff --git a/libaltaircam/CMakeLists.txt b/libaltaircam/CMakeLists.txt
index b2a8295c..d6140492 100644
--- a/libaltaircam/CMakeLists.txt
+++ b/libaltaircam/CMakeLists.txt
@@ -5,8 +5,8 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 # Using AltairCamera SDK Version: 32.13483.2018.1206
 set(LIBALTAIRCAM_VERSION "1.0.1")
@@ -25,7 +25,7 @@ exec_program(ln ARGS -s ${LIB_INSTALL_DIR}/libaltaircam.${LIBALTAIRCAM_SOVERSION
 
 ELSE(APPLE)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 IF(UNIX AND NOT WIN32)
   IF (CMAKE_SYSTEM_PROCESSOR MATCHES "armv+")
diff --git a/libapogee/CMakeLists.txt b/libapogee/CMakeLists.txt
index 383e1088..86e96b55 100644
--- a/libapogee/CMakeLists.txt
+++ b/libapogee/CMakeLists.txt
@@ -13,8 +13,8 @@ set(APOGEE_SOVERSION "3")
 IF(APPLE)
 set(CONF_DIR "/usr/local/lib/indi/DriverSupport/" CACHE STRING "Base configuration directory")
 ELSE(APPLE)
-set(CONF_DIR "/etc" CACHE STRING "Base configuration directory")
-set(UDEVRULES_INSTALL_DIR "/etc/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(CONF_DIR "${CMAKE_INSTALL_PREFIX}/etc" CACHE STRING "Base configuration directory")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/etc/udev/rules.d" CACHE STRING "Base directory for udev rules")
 ENDIF()

 # IMPORTANT: When update libapogee code to upstream. Do not forget to save apgHelper.cpp as the configuration file
diff --git a/libatik/CMakeLists.txt b/libatik/CMakeLists.txt
index 1acb75fc..4d17825f 100644
--- a/libatik/CMakeLists.txt
+++ b/libatik/CMakeLists.txt
@@ -11,7 +11,7 @@ set(FLYCAPTURE_SOVERSION "2")
 
 include(GNUInstallDirs)
 
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 IF(APPLE)
 
@@ -25,7 +25,7 @@ exec_program(ln ARGS -s ${LIB_INSTALL_DIR}/libatikcameras${ATIK_SOVERSION}.dylib
 
 ELSE(APPLE)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 IF(UNIX AND NOT WIN32)
   IF (CMAKE_SYSTEM_PROCESSOR MATCHES "armv+")
diff --git a/libfishcamp/CMakeLists.txt b/libfishcamp/CMakeLists.txt
index 72458dcd..7e59fbbb 100644
--- a/libfishcamp/CMakeLists.txt
+++ b/libfishcamp/CMakeLists.txt
@@ -13,8 +13,8 @@ include_directories( "/usr/local/include")
     ## This one is needed for Craft
 include_directories("${CMAKE_INSTALL_PREFIX}/include")
 ELSE(APPLE)
-set(FIRMWARE_INSTALL_DIR "/lib/firmware")
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(FIRMWARE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/firmware")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 ENDIF(APPLE)
 #***********************************************************
 find_package(USB1 REQUIRED)
diff --git a/libfli/CMakeLists.txt b/libfli/CMakeLists.txt
index ba2fa5a2..b04c09e6 100644
--- a/libfli/CMakeLists.txt
+++ b/libfli/CMakeLists.txt
@@ -5,7 +5,7 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 ADD_DEFINITIONS(-Wall -O2 -D__LIBUSB__)
 
diff --git a/libinovasdk/CMakeLists.txt b/libinovasdk/CMakeLists.txt
index c19ec3b9..2b51b37b 100644
--- a/libinovasdk/CMakeLists.txt
+++ b/libinovasdk/CMakeLists.txt
@@ -6,9 +6,9 @@ set(INOVASDK_SOVERSION "1")
 
 include(GNUInstallDirs)
 
-set(FIRMWARE_INSTALL_DIR "/lib/firmware")
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(FIRMWARE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/firmware")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-error")
 SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error")
diff --git a/libnncam/CMakeLists.txt b/libnncam/CMakeLists.txt
index b6665a49..546f270f 100644
--- a/libnncam/CMakeLists.txt
+++ b/libnncam/CMakeLists.txt
@@ -5,8 +5,8 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 # Using Version: 38.14088.2019.0307
 set(LIBNNCAM_VERSION "1.38.1")
@@ -25,7 +25,7 @@ exec_program(ln ARGS -s ${LIB_INSTALL_DIR}/libnncam.${LIBNNCAM_SOVERSION}.dylib
 
 ELSE(APPLE)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 IF(UNIX AND NOT WIN32)
   IF (CMAKE_SYSTEM_PROCESSOR MATCHES "armv+")
diff --git a/libqhy/CMakeLists.txt b/libqhy/CMakeLists.txt
index a0b037f5..770403c5 100644
--- a/libqhy/CMakeLists.txt
+++ b/libqhy/CMakeLists.txt
@@ -10,7 +10,7 @@ set(LIBQHY_SOVERSION "6")
 SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-error")
 SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error")
 
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 IF (APPLE)
 
@@ -36,8 +36,8 @@ exec_program(ln ARGS -s ${LIB_INSTALL_DIR}/libqhyccd.${LIBQHY_SOVERSION}.dylib $
 
 ELSE()
 
-set(FIRMWARE_INSTALL_DIR "/lib/firmware/qhy" CACHE STRING "QHY firmware installation directory")
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(FIRMWARE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/firmware/qhy" CACHE STRING "QHY firmware installation directory")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 IF(UNIX AND NOT WIN32)
  IF (CMAKE_SYSTEM_PROCESSOR MATCHES "armv+")
diff --git a/libsbig/CMakeLists.txt b/libsbig/CMakeLists.txt
index d7f35398..a976e102 100644
--- a/libsbig/CMakeLists.txt
+++ b/libsbig/CMakeLists.txt
@@ -6,7 +6,7 @@ set(SBIG_SOVERSION "4")
 
 include(GNUInstallDirs)
 
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 SET(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wno-error")
 SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-error")
@@ -24,8 +24,8 @@ exec_program(ln ARGS -s ${LIB_INSTALL_DIR}/libsbig.${SBIG_SOVERSION}.dylib ${LIB
 
 ELSE(APPLE)
 
-set(FIRMWARE_INSTALL_DIR "/lib/firmware")
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(FIRMWARE_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/firmware")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 IF(UNIX AND NOT WIN32)
   IF (CMAKE_SYSTEM_PROCESSOR MATCHES "armv+")
diff --git a/libstarshootg/CMakeLists.txt b/libstarshootg/CMakeLists.txt
index fde59906..bf5a15ec 100644
--- a/libstarshootg/CMakeLists.txt
+++ b/libstarshootg/CMakeLists.txt
@@ -5,8 +5,8 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
 include(GNUInstallDirs)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 # Using Starshoot G SDK Version: 38.14715.2019.0523
 set(LIBSTARSHOOTG_VERSION "1.38.1")
@@ -25,7 +25,7 @@ exec_program(ln ARGS -s ${LIB_INSTALL_DIR}/libstarshootg.${LIBSTARSHOOTG_SOVERSI
 
 ELSE(APPLE)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 IF(UNIX AND NOT WIN32)
   IF (CMAKE_SYSTEM_PROCESSOR MATCHES "armv+")
diff --git a/libtoupcam/CMakeLists.txt b/libtoupcam/CMakeLists.txt
index 68e4e253..25f3c716 100644
--- a/libtoupcam/CMakeLists.txt
+++ b/libtoupcam/CMakeLists.txt
@@ -5,8 +5,8 @@ LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")
 LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/../../cmake_modules/")
 include(GNUInstallDirs)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
-set(LIB_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(LIB_INSTALL_DIR "${CMAKE_INSTALL_LIBDIR}")
 
 # Using ToupCamera SDK Version: 39.15195.2019.0723
 set(LIBTOUPCAM_VERSION "1.39.0")
@@ -25,7 +25,7 @@ exec_program(ln ARGS -s ${LIB_INSTALL_DIR}/libtoupcam.${LIBTOUPCAM_SOVERSION}.dy
 
 ELSE(APPLE)
 
-set(UDEVRULES_INSTALL_DIR "/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
+set(UDEVRULES_INSTALL_DIR "${CMAKE_INSTALL_PREFIX}/lib/udev/rules.d" CACHE STRING "Base directory for udev rules")
 
 IF(UNIX AND NOT WIN32)
   IF (CMAKE_SYSTEM_PROCESSOR MATCHES "armv+")
